src/ui/default-prop-types.js

```diff
@@ -28,10 +28,11 @@ export const importSourceShape = PropTypes.shape(importSourceProps);
 
 export const clientConfigProps = {
   clientConfig: PropTypes.shape({
-    cdnRootUrl: PropTypes.string.isRequired
-  }).isRequired,
-  importSources: PropTypes.arrayOf(importSourceShape),
-  cookieConsentName: PropTypes.string.isRequired
+    cdnRootUrl: PropTypes.string.isRequired,
+    disabledFeatures: PropTypes.arrayOf(PropTypes.string).isRequired,
+    importSources: PropTypes.arrayOf(importSourceShape).isRequired,
+    consentCookieName: PropTypes.string.isRequired
+  }).isRequired
```

Comment:
This data structure was just wrong and got me a red console in the browser dev tools. It should be correct now.

Reply:
Indeed it was wrong 😶‍🌫️